assets/helpers/contributions.js

```diff
@@ -56,7 +56,7 @@ const config: Config = {
     default: 75,
   },
   MONTHLY: {
-    min: 5,
+    min: 2,
```

Comment:
I would change the `minInWords` field as well.

Reply:
👍